catalog/services_state_test.go

```diff
@@ -3,21 +3,23 @@ package catalog
 import (
 	"encoding/json"
 	"fmt"
+	"net/http"
 	"regexp"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/Nitro/sidecar/service"
+	"github.com/jarcoal/httpmock"
```

Comment:
Sidecar already has an HTTP mocking library that it uses for other tests. I like the one you are using better. But we should either delete the one I wrote and fix the other tests, or use the Sidecar one here. Code is [here](https://github.com/newrelic/sidecar/tree/master/mockhttp)